src/wagtail_ai/migrations/0002_populate_default_prompts.py

```diff
@@ -0,0 +1,48 @@
+# Generated using Django 4.2.7 on 2023-12-01 09:35
+
+from django.conf import settings
+from django.db import migrations
+
+import wagtail_ai
+
+
+def set_default_ai_prompts(apps, schema_editor):
+    """
+    Loop through each default prompt in settings and
+    create instances in the database.
+    """
+    Prompt = apps.get_model("wagtail_ai", "Prompt")
+
+    default_prompts = (
+        wagtail_ai.DEFAULT_PROMPTS if hasattr(wagtail_ai, "DEFAULT_PROMPTS") else []
+    )
```

Comment:
**Issue:** Should the default prompts be hard-coded in the migration instead? I feel uneasy that you have dynamic values in a migration. I think we should get rid of the prompts settings.